test-cases/tests/installation/a21-verify-maintenance-and-backup-windows.md

```diff
@@ -0,0 +1,30 @@
+---
+targets:
+- 2.3.0
+estimate: 30m
+---
+
+# A21 - Verify maintenance and backup windows
+
+## AWS Infrastructure Steps
+The following steps are valid for a cluster provisioned in AWS backed by AWS RDS, Elasticache and S3 resources
+
+1. After RHMI install check for `cloud-resources-aws-strategies` config map in RHMI operator ns
+    - If this config map is present make note of the values in the `CreateStrategy` for both `Redis` and `Postgres`
+2. For each Resource Created in AWS RDS and Elasticache 
+    - Note the maintenance and backup/snapshot windows 
```

Comment:
Can we be more explicit here? e.g. in the AWS console this can be seen through the tab <tab name> on an RDS instance